test/unit/specs/utils/attribution-html.spec.ts

```diff
@@ -0,0 +1,63 @@
+import Vuei18n from 'vue-i18n'
+
+import type { AttributableMedia } from '~/utils/attribution-html'
+import { getAttribution } from '~/utils/attribution-html'
+import enMessages from '~/locales/en.json'
+
+const i18n = new Vuei18n({
+  locale: 'en',
+  fallbackLocale: 'en',
+  messages: { en: enMessages },
+})
+
+const mediaItem: AttributableMedia = {
+  title: 'Title',
+  foreign_landing_url: 'https://foreign.landing/url',
+  creator: 'Creator',
+  creator_url: 'https://creator/url',
+  license: 'pdm',
+  license_version: '1.0',
+  license_url: 'https://license/url',
+}
+
+describe('getAttribution', () => {
+  it.each`
+    sendI18n | attributionText
+    ${true}  | ${'"Title" by Creator is marked with Public Domain Mark 1.0 .'}
```

Comment:
Can we remove the space before the final period in the attribution text?

Reply:
This is not an actual space, we're comparing the textual content of the HTML with this string (there are actually `<img>` tags with license icons there). 

I've added an extra test case in 9a66217 to confirm that there are no stray spaces in the output.